proto/poktroll/migration/tx.proto

```diff
@@ -111,3 +113,24 @@ message MsgClaimMorseAccountResponse {
   int64 claimed_at_height = 3;
 }
 
+// TODO_UPNEXT(@bryanchriswhite, #1034):
+// - Refactor to consistent naming convention
+// - Add comments describing the purpose of each message/field
+
+message MsgClaimMorseApplication {
+  option (cosmos.msg.v1.signer) = "shannonDestAddress";
+  string                   shannonDestAddress = 1;
+  string                   morseSrcAddress    = 2;
+  string                   morseSignature     = 3;
+  cosmos.base.v1beta1.Coin stake              = 4 [(gogoproto.nullable) = false];
+  string                   serviceConfig      = 5;
+}
+
+message MsgClaimMorseApplicationResponse {
```

Comment:
Related to the point above, should we embed `poktroll.application.Application application = 1;` (a shared structure) in here to align with `MsgStakeApplicationResponse`?

Reply:
Nice catch! :pray: